include/picongpu/fields/FieldManipulator.kernel

```diff
@@ -95,6 +95,7 @@ namespace detail
             uint32_t const workerIdx = threadIdx.x;
 
             auto const numGuardSuperCells = mapper.getGuardingSuperCells();
+            DataSpace< simDim > guardCells(numGuardSuperCells * SuperCellSize::toRT());
```

Comment:
Shouldn't it be formatted as  `DataSpace< simDim > guardCells( numGuardSuperCells * SuperCellSize::toRT() );` ? (spaces with brackets)

Reply:
Yes, but we don't *partly* update legacy code for consistency and to avoid bikeshedding.

Just in new files or when files are updated at once (via automated solutions that we still have to find).

Reply:
Yes I will update the spaces, this file is already formatted in the new style.